npm/ng-packs/packages/account/src/lib/guards/extensions.guard.ts

```diff
@@ -44,3 +47,28 @@ export class AccountExtensionsGuard implements IAbpGuard {
     );
   }
 }
+
+export const AccountExtensionsGuardFn = () => {
```

Comment:
it should be resolve instead of  guard.
it is guard because there was no resolve function

https://angular.io/api/router/Resolve